frontend/src/services/Api.ts

```diff
@@ -49,6 +49,11 @@ export interface Response<T> {
   data: T;
 }
 
+/** Get internal Kiali proxy if exists */
+const getKialiProxy = (): string | null => {
```

Comment:
When I see the word "proxy" I think of an HTTP proxy that sits in front of some other server. But I don't think that's what this means.

Can you add some comments here about what this proxy is? Because right now it isn't clear. Is this an HTTP proxy or something else?

Also, can you explain WHERE this environment variable is set? If I want to set "KIALI_PROXY" environment variable, where is that done? Is that only for, say, a development environment (i.e. "yarn start" where I set the environment variable in my own command line window?) or is this also going to be set when deployed inside the cluster? If so, who sets this environment variable?

In short, this is confusing to people not familiar with this stuff. Having a few lines of comments here would help people to understand what this proxy is and how it is meant to be set.

Reply:
I have renamed to API proxy and added a comment that this is only defined in some platforms like Openshift console and not application for standalone Kiali application